src/main/kotlin/config/Configs.kt

```diff
@@ -61,8 +61,12 @@ interface BitcoinConfig {
     val blockStoragePath: String
     //Depth of transactions in BTC blockchain
     val confidenceLevel: Int
-    //BTC node host name
-    val host: String
+    //BTC node hosts
+    val hosts: String
+
+    companion object {
+        fun extractHosts(bitcoinConfig: BitcoinConfig) = bitcoinConfig.hosts.replace(" ", "").split(",")
```

Comment:
I think it is better to use a different approach. I mean something closer to `commons-configurations` implementation should be used so that we could have `List<String>` or `String[]`  in our config class and parse the value into the required type at the config loading phase.

Reply:
cfg4k is not very flexible to do this. I think we should move configurations files to JSON/YAML format or try another configuration framework.